schemata/pdf_manager_template.json

```diff
@@ -3,7 +3,6 @@
   "id": "/schemata/pdf_manager_template",
 
   "title": "PDF Manager Template",
-  "stability": "prototype",
```

Comment:
It'll change a lot of files in 2 projects, I think it's better doing it in another PR/ticket.